python/psautohint/autohint.py

```diff
@@ -30,15 +30,18 @@
 
 from __future__ import print_function, absolute_import
 
-import sys
 import os
 import re
+import shutil
+import sys
 import time
-import plistlib
 import traceback
-import shutil
 
-from psautohint import psautohint
+from fontTools.ttLib import TTFont, TTLibError, getTableModule
+
+from .psautohint import autohint, autohintmm
```

Comment:
I personally prefer `import foo` and `foo.Bar` style  to `from .foo import Bar` as it makes it a bit clear at the calling site where `Bar` is coming from, and keeps the global namespace a bit tidier.

Reply:
I prefer it this way because it makes it clear right at the top which pieces the file is using. Doing a general import at the top and only using parts of it downstream makes the imports opaque IMO.